packages/cli/src/models/compiler/Compiler.js

```diff
@@ -0,0 +1,46 @@
+import { exec } from 'child_process'
+import { FileSystem, Contracts, Logger } from 'zos-lib'
+import TruffleConfig from '../initializer/truffle/TruffleConfig'
+import SolidityProjectCompiler from './solidity/SolidityProjectCompiler'
+
+const log = new Logger('Compiler')
+
+export default {
+  async call() {
+    return this._isTruffleProject()
+      ? this.compileWithTruffle()
```

Comment:
If we don't call this method outside the class, let's add the `_` at the beginning of the function name for consistency.
```suggestion
      ? this._compileWithTruffle()
```

Reply:
I left those as is in case someone would like to compile with a particular compiler besides using Truffle or not